registry/configuring-registry-operator.adoc

```diff
@@ -7,31 +7,36 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 [id="image-registry-on-cloud"]
-ifndef::openshift-dedicated,openshift-rosa[]
+ifndef::openshift-dedicated,openshift-rosa,openshift-rosa-hcp[]
 == Image Registry on cloud platforms and OpenStack
-endif::openshift-dedicated,openshift-rosa[]
-ifdef::openshift-dedicated,openshift-rosa[]
+endif::openshift-dedicated,openshift-rosa,openshift-rosa-hcp[]
+ifdef::openshift-dedicated,openshift-rosa,openshift-rosa-hcp[]
 == Image Registry on {product-title}
-endif::openshift-dedicated,openshift-rosa[]
+endif::openshift-dedicated,openshift-rosa,openshift-rosa-hcp[]
 
 The Image Registry Operator installs a single instance of the {product-registry}, and manages all registry configuration, including setting up registry storage.
 
-ifndef::openshift-dedicated,openshift-rosa[]
+ifndef::openshift-dedicated,openshift-rosa,openshift-rosa-hcp[]
 [NOTE]
 ====
 Storage is only automatically configured when you install an installer-provisioned infrastructure cluster on AWS, Azure, GCP, {ibm-name}, or OpenStack.
 
 When you install or upgrade an installer-provisioned infrastructure cluster on AWS, Azure, GCP, {ibm-name}, or OpenStack, the Image Registry Operator sets the `spec.storage.managementState` parameter to `Managed`. If the `spec.storage.managementState` parameter is set to `Unmanaged`, the Image Registry Operator takes no action related to storage.
 ====
-endif::openshift-dedicated,openshift-rosa[]
+endif::openshift-dedicated,openshift-rosa,openshift-rosa-hcp[]
 
-After the control plane deploys, the Operator creates a default `configs.imageregistry.operator.openshift.io` resource instance based on configuration detected in the cluster.
+After the control plane deploys in the management cluster, the Operator creates a default `configs.imageregistry.operator.openshift.io` resource instance based on configuration detected in the cluster.
 
 If insufficient information is available to define a complete `configs.imageregistry.operator.openshift.io` resource, the incomplete resource is defined and the Operator updates the resource status with information about what is missing.
 
+ifdef::openshift-rosa-hcp[]
+The Image Registry Operator runs in the `HOSTED_CLUSTERS_NAMESPACE` of the management cluster, and manages the registry instance in openshift-image-registry namespace of the cluster. All configuration and workload resources for the registry reside in that namespace.
```

Comment:
Same as above

Reply:
@EricPonvelle Do I need to add this also here as suggested below:
ifdef::openshift-rosa,openshift-rosa-hcp[]
ifdef::openshift-rosa-hcp[]
The Image Registry Operator runs in the `CONTROL_PLANE_NAMESPACE` of the management cluster, and manages the registry instance in openshift-image-registry namespace of the cluster. All configuration and workload resources for the registry reside in that namespace.
endif::openshift-rosa-hcp[]
ifdef::openshift-rosa[]
The Image Registry Operator runs in the `openshift-image-registry` namespace, and manages the registry instance in that location as well. All configuration and workload resources for the registry reside in that namespace.
endif::openshift-rosa[]
endif::openshift-rosa,openshift-rosa-hcp[]